about.html

```diff
@@ -0,0 +1,128 @@
+                <p>Made in Family...</p>
+                <h1>
+                    WHAT WE <br> ARE ABOUT
+                </h1>
+            </div>
+            <div class="hero-info">
+                <p>At Fratello Pizzas we don’t just make pizza. We make people happy. Fratello was built on the belief that sharing 
+                    pizza with family should be special, and we carry that belief into everything we do. With more than 30
+                    years of experience cooking under our belts.</p>
+            </div>
+            <div class="hero-date">
+                <h2>Do you want to know us closer? Write us directly and you can become one integrant of the Fratello family...</h2>
+                <a href="mailto:fratellopizzas.sc@gmail.com" target="_blank">fratellopizzas.sc@gmail.com</a>
+            </div>
+        </section>
+    </header>
+    <section class="fratello-logo" id="logo">
+        <h2>Fratello Pizzas Logo</h2>
+        <hr />
+        <p>The original Fratello's logo featured a semi-realistic looking an actual pizza, and the Fratello's Pizza wordmark
+            usually being placed to either side of the pizza.<br>Currently The triangles on the logo represent the brotherhood and the family.
+         Originally, Fratello's had planned to add a triangle for each new restaurant opened.
+        </p>
+        <div class="logo-middle"><img src="./assets/img/fratello-logo-negro-horizontal.png"></div>
+    </section>
+    <section class="story" id="story">
+        <h2>A quick look to our Story...</h2>
+        <hr />
+        <p>What Fratello Pizzas is today began in 2020 with the dream of two local brothers of San Cristobal city, both with a big sensation of love for cooking,<br> with not much money in their pockets but with a big motivation and their family support.</p>
+        <div class="story-cards-container">
+            <article class="story-card">
+                <h4>2020</h4>
+                <p>They sell their first 4 pizzas for a group of friends in a meeting, all people loved those pizzas and the voice starts spread on the city. They start only doing delivery because of the quarantine.</p>
+            </article>
+            <article class="story-card2">
+                <h4>2021</h4>
+                <p>Fratellos's brothers open their first local store and it was an amazing success for them. They started offering jobs for the locals and people of the city keeps in love with their pizzas</p>
+            </article>
+        </div>
+    </section>
+    </section>
+    <div class="locations" id="locations">
+        <h2>Location</h2>
+        <hr />
+        <div class="direction">
+            <i class="fa-solid fa-shop"></i>
+            <div class="direction-info">
+                <h4>San Cristobal</h4>
+                <p>Bolivar Main Street, C.C Los Ilustres, Level Feria, Local N/A,.<br>
+                    Phone:+58(0414) 746-4054</p>
+            </div>
+        </div>
+    </div>
```

Comment:
- [ ] On mobile view the information can't be seen clearly. You can increase a little bit the font-size and on the desktop view increase the padding on both pages.